### PR TITLE
feat(elements|ino-chip): add slot for icons and mark ino-icon prop as deprecated

### LINKED ISSUE #188: ino-chip: Use slots instead of `ino-icon` property
If you want to use an `ino-icon` inside of an `ino-chip`, you have to pass the icon name per the `ino-icon` property. This limits the flexibility of the icon because you cannot use all props of the `ino-icon` component (e.g. `src`). We should provide a slot for the icon, so we can use the whole API of the `ino-icon` component.

This solution is already implemented e.g. in the `ino-button`.

### PR DESCRIPTION
**Pull request by P1NHE4D**
✨ **Features:**
* Removed the `ino-icon` property
* Added a slot with the name `ino-icon-leading
* Adjusted the `ino-icon` and `ino-chip` stories accordingly

🎟️ **Closes #188**

### CODE REVIEW

**Comment by janivo** [comment]
Maybe we should support both variants for now and print a warning that this feature (using the `ino-icon` prop) will be deprecated in the future. Otherwise, we might have to release a new major version again in such a short period of time. What do you think?

**Comment by P1NHE4D** [comment]
> Maybe we should support both variants for now and print a warning that this feature (using the `ino-icon` prop) will be deprecated in the future. Otherwise, we might have to release a new major version again in such a short period of time. What do you think?

This is a great idea. I'll add it right away.

**Review by JCofman — APPROVED** [review]
LGTM :)

File: packages/elements/src/components/ino-chip/ino-chip.tsx [hunk 1]
```diff
@@ -26,7 +26,13 @@ export class Chip implements ComponentInterface {
   @Prop() inoFill?: ChipSurface = 'solid';
 
   /**
-   * If enabled, prepends the slotted icon to the chip
+   * Prepends an icon to the chip label.
+   * @deprecated This property is deprecated and will be removed with the next major release. Instead, use the ino-icon-leading slot.
```
Comment (janivo) [inline packages/elements/src/components/ino-chip/ino-chip.tsx:30]: I would suggest that we also check if the `inoIcon` property is used initially (e.g. in `componentDidMount`) and print a warning with `console.warn` with the deprecated message above. This way we make sure that everyone is informed even if he's not checking the documentation regularly.

**Review by janivo — CHANGES_REQUESTED** [review]

File: packages/elements/src/components/ino-chip/ino-chip.tsx (hunk 1 above)
Comment (P1NHE4D) [reply packages/elements/src/components/ino-chip/ino-chip.tsx:30]: Good idea

**Review by janivo — APPROVED** [review]

**Pull request merged**